Functions/Quantification/xASL_quant_CBF.m

```diff
@@ -1,6 +1,6 @@
-function [ScaleImage, CBF] = xASL_quant_SinglePLD(PWI, M0_im, imSliceNumber, x, bUseBasilQuantification)
-%xASL_quant_SinglePLD % Perform a multi-step quantification
-% FORMAT: [ScaleImage[, CBF]] = xASL_quant_SinglePLD(PWI, M0_im, imSliceNumber, x)
+function [ScaleImage, CBF] = xASL_quant_CBF(PWI, M0_im, imSliceNumber, x, bUseBasilQuantification)
```

Comment:
This name is not correct anymore, since we now also output ATT, Tex, and perhaps more in the feature, from this function. Should `xASL_quant_ASL` be more correct?

I know that our pipeline is called "ExploreASL" but we can also process other data than ASL :)

Reply:
Sorry, a mistake. I'm correcting that.